Changelog

```diff
@@ -1,4 +1,81 @@
+    - BUGFIX: Remove artificial max limits from `Torrent Queueing` related options (Chocobo1)
+    - BUGFIX: Preserve `skip hash check` when there is no metadata (glassez)
+    - BUGFIX: Fix DHT/PeX/LSD status when it is globally disabled (Kacper Michajłow)
+    - BUGFIX: Fix rate calculation when interval is too low (glassez)
+    - BUGFIX: Add tooltip message when system tray icon isn't available (Chocobo1)
+    - BUGFIX: Improve sender field in mail notifications (Dmitry Vodopyanov)
+    - BUGFIX: Fix "Add torrent dialog" spill-over on smaller screens (Chocobo1)
+    - BUGFIX: Fix peer count issue when tracker responds with zero figure (summer)
+    - BUGFIX: Don't merge trackers by default (glassez)
+    - BUGFIX: Don't inhibit system sleep/auto shutdown for torrents stuck at downloading metadata (summer)
+    - BUGFIX: Allow to pause a checking torrent from context menu (summer)
+    - BUGFIX: Allow to use subnet notation in reverse proxy list (Chocobo1)
+    - BUGFIX: Fine tune translations loading for Chinese locales (sledgehammer999)
+    - BUGFIX: Fix torrent content checkboxes not updated properly (Chocobo1)
+    - BUGFIX: Correctly load state of `Use another path for incomplete torrents` in Watched folders (glassez)
+    - BUGFIX: Fix wrong count of errored trackers (Chocobo1)
+    - WEBUI: Allow blank lines in multipart form-data input (Aleksandr Cupacenko)
+    - WEBUI: Make various dialogs resizable (Chocobo1)
+    - WEBUI: Fix wrong v2 hash string displayed (Chocobo1)
+    - WEBUI: WebAPI: return correct status (Requi)
+    - WEBUI: Fix empty selection in language combobox (Chocobo1)
+    - WEBUI: Store WebUI port setting in human readable number (Chocobo1)
+    - WEBUI: Add support for exporting .torrent (Tom Piccirello)
+    - WEBUI: WebAPI: Add endpoint to set speed limit mode (glassez)
+    - WEBUI: Improve progress bar rendering (Mike Lei)
+    - WEBUI: Add transfer list refresh interval settings (summer)
+    - WEBUI: Use natural sort (Chocobo1)
+    - WEBUI: Apply i18n translation only to built-in WebUI (Chocobo1)
+    - WEBUI: Alert when HTTPS settings are incomplete (Chocobo1)
+    - WEBUI: Handle drag and drop events (Chocobo1)
+    - WEBUI: Fix wrong behavior for shutdown action (Chocobo1)
+    - WEBUI: Don't disable combobox for file priority (Chocobo1)
+    - RSS: Increase limit of maximum number of articles per feed (summer)
+    - WINDOWS: Fix `Open destination folder` delay on Windows (Andrew)
+    - WINDOWS: NSIS: Update Russian, Estonian, Japanese, Dutch, Portuguese BR, German and Indonesian translations (Andrei Stepanov, Priit Uring, maboroshin, Thomas De Rocker, Ícaro, schnurlos, Faisal A. F. Rahman)
+    - LINUX: Mark as single window app in .desktop file (Nicolas Fella)
+    - LINUX: Add Dockerfile (Amanuense-del-diavolo, Tom Piccirello, Chocobo1)
+    - LINUX: Remove option of using icons from system theme (now-im)
+    - MACOS: Don't autofill background color on macOS (Tyler Dunn)
```

Comment:
You've got me listed as my name in this commit and my handle in the other. I don't mind either, but I would at least prefer it to be consistent (let's go with my handle)

Reply:
I suppose it doesn't mean too much to the user it as is either, maybe something along the lines of "Fix background color behind properties widget"

Reply:
>You've got me listed as my name in this commit and my handle in the other. I don't mind either, but I would at least prefer it to be consistent (let's go with my handle)

I use a script to pull and format the commits from git history. The name is extracted from the commit author name. Maybe you made commits from 2 differently configured git clients?

>"Fix wrong background color behind properties widget"

:+1:

Reply:
I might have done one via the web interface if I was at work. I'll try to keep it consistent in future commits. Good to know, thanks.